Makefile

```
@@ -418,7 +422,42 @@ ensure-backend-volumes: ## Create and permission backend volume directories
+	if [ -n "$$missing" ]; then \
+		echo "$(RED)Error: the following required variables are missing or empty in $(ENV_FILE):$(NC)"; \
+		for var in $$missing; do \
+			echo "  - $$var"; \
+		done; \
+		echo ""; \
+		echo "$(YELLOW)Set them in $(ENV_FILE) and try again.$(NC)"; \
+		echo "Tip: 'make generate-langflow-password' can auto-generate Langflow credentials."; \
+		exit 1; \
+	fi
+
+generate-langflow-password: ## Auto-generate and append Langflow superuser credentials to .env
```

checkmake (0.3.2)

[warning] Target body for "generate-langflow-password" exceeds allowed length of 5 lines (14).

(maxbodylength)